code/datums/craft/recipes/weapon.dm

```diff
@@ -220,6 +220,18 @@
 		list(QUALITY_ADHESIVE, 10, "time" = 5)
 	)
 
+/datum/craft_recipe/weapon/armknife
+	name = "Wrist Shank"
+	result = /obj/item/organ_module/active/simple/shanker
+	steps = list(
+		list(CRAFT_MATERIAL, 2, MATERIAL_STEEL, "time" = 10),
```

Comment:
```suggestion
		list(CRAFT_MATERIAL, 8, MATERIAL_STEEL, "time" = 10),
```
So we don't duplicate materials.
There is a slight plastic cost (because cable coils have a little plastic and steel) but we can disregard it